test/global-rxjs.ts

```diff
@@ -1,4 +1,5 @@
 /**
+ *
```

Comment:
I don't understand why do we need this file. Could you provide some insight? This file seems to just reexport things which means that it should be unnecessary? Could you explain what issue you are trying to solve?